src/utilities/__tests__/astFromValue-test.js

```diff
@@ -33,10 +33,14 @@ describe('astFromValue', () => {
       { kind: 'BooleanValue', value: false }
     );
 
-    expect(astFromValue(null, GraphQLBoolean)).to.deep.equal(
+    expect(astFromValue(undefined, GraphQLBoolean)).to.deep.equal(
       null
     );
 
+    expect(astFromValue(null, GraphQLBoolean)).to.deep.equal(
```

Comment:
@leebyron I'm little confused by this.
What is correct result for `astFromValue(null, new GraphQLNonNull(*))`?

In `astFromValue` implementation there is this comment at line 74:
> Note: we're not checking that the result is non-null.
> This function is not responsible for validating the input value.
So, it is correct to break this logic?